metron-interface/metron-alerts/.nvmrc

```diff
@@ -1 +1 @@
-v9.11.1
+v10.15.1
```

Comment:
Travis uses nvm and .nvmrc file. Whit this we can pin down the node version both locally and on Travis.